lib/timers.js

```diff
@@ -232,6 +231,12 @@ function listOnTimeout() {
   } else if (list === refedLists[msecs]) {
     delete refedLists[msecs];
   }
+
+  // do not close reused handler which is unrefed in setInterval
+  if (this.owner && this.owner._repeat)
```

Comment:
At first, I worried the case of using `setTimeout` but I found that it doesn't matter. 
Right, `this.owner._repeat` is not necessary. Fixed.